openassessment/xblock/test/test_peer.py

```diff
@@ -286,7 +286,6 @@ def test_turbo_grading(self, xblock):
         self.assertIsNotNone(peer_response)
         self.assertNotIn(submission["answer"]["parts"][0]["text"].encode('utf-8'), peer_response.body)
         self.assertNotIn(submission["answer"]["parts"][1]["text"].encode('utf-8'), peer_response.body)
-        self.assertIn("Peer Assessments Complete", peer_response.body)
```

Comment:
In the interest of not reducing test coverage, could this be changed to look for `You have successfully completed all of the required peer assessments for this assignment` instead of being removed entirely?